app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -121,7 +126,8 @@ def self.params_for_create(ems)
   end
 
   def self.raw_create_cloud_subnet(ext_management_system, options)
-    cloud_tenant = options.delete(:cloud_tenant)
+    cloud_tenant_id = options.delete(:cloud_tenant_id)
+    cloud_tenant = CloudTenant.find_by(:id => cloud_tenant_id) if cloud_tenant_id
```

Comment:
@agrare it looks like this change may have broken a test case but I'm not sure how to resolve it

Reply:
@DavidResende0 you'll need to update the params passed in the spec to match what you updated here so: `ems_network.create_cloud_subnet(:cloud_tenant_id => tenant.id)`